model-optimizer/extensions/back/ResultRename.py

```diff
@@ -0,0 +1,35 @@
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.back.TopKNormalizer import TopKNormalizer
+from mo.back.replacement import BackReplacementPattern
+from mo.graph.graph import Graph
+
+
+class ResultRename(BackReplacementPattern):
+    enabled = False
```

Comment:
Please, leave a comment describing the purpose of this transformation and why it is disabled by default.